ci/licenses_golden/licenses_flutter

```diff
@@ -656,6 +656,8 @@ FILE: ../../../flutter/shell/common/shell_test_platform_view_vulkan.h
 FILE: ../../../flutter/shell/common/shell_unittests.cc
 FILE: ../../../flutter/shell/common/skia_event_tracer_impl.cc
 FILE: ../../../flutter/shell/common/skia_event_tracer_impl.h
+FILE: ../../../flutter/shell/common/sk_trace_memory_dump_json.cc
+FILE: ../../../flutter/shell/common/sk_trace_memory_dump_json.h
```

Comment:
These changes and related are actually from #22559 since that's been helpful to check whether this is working or not - they'd be landed separately in that patch and will disappear from this one.